videoretrieval/tests/loss_tests.py

```diff
@@ -50,19 +50,21 @@ def test_loss_calculation_with_different_embeddings(self):
         ])]
 
         loss = bidirectional_max_margin_ranking_loss(
-            mock_perfect_video_embeddings,
-            mock_perfect_text_embeddings,
-            mock_mixture_weights,
-            mock_missing_experts,
+            build_similarity_matrix(
```

Comment:
Assign this matrix to a variable as follows:

similarity_matrix = build_similarity_matrix(
  mock_perfect_video_embeddings,
  mock_perfect_text_embeddings,
  mock_mixture_weights,
  mock_missing_experts);

loss = bidirectional_max_margin_ranking_loss(similarity_matrix, 1.0)


ditto for other usages.